docs/package.json

```diff
@@ -13,7 +13,8 @@
     "@stackblitz/sdk": "^1.9.0",
     "@vueuse/core": "^10.5.0",
     "radix-vue": "link:../packages/radix-vue",
-    "vue": "^3.3.8"
+    "vue": "^3.3.8",
+    "vue-sonner": "^1.0.2"
```

Comment:
Let's not use `vue-sonner` as an example. Instead let's use the `Toast` componennt

Reply:
Ok, I will change it and give you a response ❤️